frontend/app/services/zuora/ZuoraRestService.scala

```diff
@@ -0,0 +1,72 @@
+package services.zuora
+
+import com.gu.lib.okhttpscala._
+import com.gu.membership.util.Timing
+import com.gu.membership.zuora.ZuoraApiConfig
+import com.gu.monitoring.{AuthenticationMetrics, StatusMetrics}
+import com.squareup.okhttp.OkHttpClient
+import com.squareup.okhttp.Request.Builder
+import model.Zuora.Feature
+import monitoring.TouchpointBackendMetrics
+import play.api.libs.concurrent.Execution.Implicits._
+
+import scala.concurrent.Future
+import play.api.libs.json._
+import play.api.libs.json.Reads._
+import play.api.libs.functional.syntax._
+
+import scala.util.{Failure, Success, Try}
+
+class ZuoraRestService(config: ZuoraApiConfig) {
+
+  import ZuoraRestResponseReaders._
+
+  private val client = new OkHttpClient
+
+  val metrics = new TouchpointBackendMetrics with StatusMetrics with AuthenticationMetrics {
+    val backendEnv = config.envName
+    val service = "ZuoraRestClient"
+
+    def recordError() {
+      put("error-count", 1)
+    }
+  }
+
+  private def get(uri: String) = client.execute(new Builder()
+    .addHeader("apiAccessKeyId", config.username)
```

Comment:
Are the "apiAccessKeyId" and "apiSecretAccessKey" really the username and password - not any special token?!

https://knowledgecenter.zuora.com/BC_Developers/REST_API/A_REST_basics#Authentication